script/build.js

```diff
@@ -71,6 +72,8 @@ if (isHerokuBuild) {
   applyHerokuOptions(options);
 }
 
+smith.use(createResourceConfig(options));
+
```

Comment:
I added the middleware registration here so that it's processed into the `metadata` in time for the pages to use if they choose to.